packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -74,7 +74,7 @@ graphql`
 const query = graphql`
   query ActivityLibraryQuery {
     viewer {
-      availableTemplates(first: 200) @connection(key: "ActivityLibrary_availableTemplates") {
+      availableTemplates(first: 2000) @connection(key: "ActivityLibrary_availableTemplates") {
```

Comment:
-1 can you also increase it here? https://github.com/ParabolInc/parabol/blob/badbd0ffd7440cb43ad45fef79c9b40c09fa82a6/packages/client/components/ActivityLibrary/CreateNewActivity/CreateNewActivity.tsx#L124

Reply:
Fixed!